src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,29 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
`post.user` might be `undefined`. In this case, we'll get an error inside `UserInfo` component

Use conditional rendering to not render `UserInfo` if `post.user` is not defined